metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/files/bro_index.template

```diff
@@ -238,6 +238,538 @@
         "qtype_name": {
           "type": "string",
           "index": "not_analyzed"
+        },
+        "analyzer": {
+          "type": "string",
+          "index": "not_analyzed"
+        },
+        "failure_reason": {
```

Comment:
The refactor of this template that you suggest seems scary to me without some comments.  Duplicate names seem also like something that should be called out in comments.  There is some useful information here that can only be found in this PR and won't live-on with the code, unfortunately.

But at least it looks like you have one test record per new log type, so the integration tests should fail if a refactor of the index template goes badly.  **I think?**